NorthstarDedicatedTest/squirrel.cpp

```diff
@@ -241,6 +243,17 @@ template <ScriptContext context> SQInteger SQPrintHook(void* sqvm, char* fmt, ..
 			buf[charsWritten - 1] = '\0';
 
 		spdlog::info("[{} SCRIPT] {}", GetContextName(context), buf);
+
+		if (IsDedicated())
+		{
+			if (CVar_sv_rcon_sendlogs->GetBool())
+			{
+				char sendbuf[1024] {};
+
+				snprintf(sendbuf, sizeof(sendbuf), "[%s SCRIPT] %s", GetContextName(context), buf);
+				g_pRConServer->Send(sendbuf);
```

Comment:
why not just do this our spdlog sink to capture all logs, rather than doing it manually for different log sources?

Reply:
> why not just do this our spdlog sink to capture all logs, rather than doing it manually for different log sources?

Where is your sink, is it by any change 'SourceConsoleSink::sink_it_()'?

Reply:
oh nvm actually, you'd have to create a new spdlog sink
SourceConsoleSink is the console sink used for the vgui console, shouldn't be super hard to modify the code used for initializing it to make a new sink (https://github.com/R2Northstar/NorthstarLauncher/blob/main/NorthstarDedicatedTest/sourceconsole.cpp#L39)

Reply:
I will take a look into creating a sink dedicated to this.